tests/providers/google/cloud/transfers/test_gcs_to_bigquery.py

```diff
@@ -44,10 +44,13 @@ def test_execute_explicit_project(self, bq_hook):
 
         bq_hook.return_value.get_job.return_value.result.return_value = ('1',)
 
-        operator.execute(None)
+        result = operator.execute(None)
+
+        self.assertEqual(result, '1')
```

Comment:
```suggestion
        assert result == '1'
```

Pytest-style assertions are preferred in new code.